connector/connect/common/src/main/protobuf/spark/connect/expressions.proto

```diff
@@ -77,9 +77,7 @@ message Expression {
       int32 year_month_interval = 20;
       int64 day_time_interval = 21;
 
-      Array array = 22;
-      Struct struct = 23;
-      Map map = 24;
+      DataType typed_null = 22;
```

Comment:
where do we read this field at the server side?

Reply:
fix in https://github.com/apache/spark/pull/39060